doc-content/enterprise-only/processes/interacting-with-processes-viewing-process-instances-variables-advanced-filters-proc.adoc

```diff
@@ -1,7 +1,7 @@
-[id='interacting-with-processes-process-instances-special-filter-proc']
-= Defining special filters for process instances
+[id='interacting-with-processes-viewing-process-instances-variables-advanced-filters-proc']
```

Comment:
I expect this ID isn't used elsewhere, but just to be safe, did you check to make sure? If you find any, they will need to be updated.